RecoParticleFlow/PFProducer/plugins/PFBlockProducer.cc

```diff
@@ -347,6 +347,7 @@ void PFBlockProducer::fillDescriptions(edm::ConfigurationDescriptions& descripti
     psd.addOptional<bool>("SuperClusterMatchByRef", false);
     desc.addVPSet("linkDefinitions", psd, vpset);
   }
+  desc.addOptional<bool>("useNuclear", false);
```

Comment:
I am not sure to follow this change. Is the parameter `useNuclear` actually used in `PFBlockProducer` ?

Reply:
This change was triggered by the errors in the HI relvals (in where they add the parameter to the producer). But it is a good question, I didn't check which producers is the one actually using it

Reply:
removed the useNuclear in 50d3e5865bc74020f452a19bf4e852a689bfd4fe